src/Flame/Renderer/Hook.purs

```diff
@@ -0,0 +1,66 @@
+-- | Snabbdom VNode hooks
+module Flame.Renderer.Hook where
```

Comment:
I've thought that Hook is advanced rendering feature. Okay, will move that to HTML namespace, but I'm not sure that we should reexport it.

Reply:
Oh I see your point. I was thinking in terms of node data, but since it is directly changing the nodes themselves might be worth to keep it on `Renderer`. Up to you!

Reply:
However, as Attribute is mostly imported with `qualified`, I'm doing it.

Reply:
Oh, I haven't seen your 'up to you message'. 
The point also is that Renderer could be considered as an internal module, and mixing that with hooks... 

I think I'll revert my mind to Renderer namespace.